buildenv/jenkins/JenkinsfileBase

```diff
@@ -624,29 +624,33 @@ def buildTest() {
 
 		try {
 			//get pre-staged jars from test.getDependency build before test compilation
-			timeout(time: 2, unit: 'HOURS') {
-				copyArtifacts fingerprintArtifacts: true, projectName: "test.getDependency", selector: lastSuccessful(), target: 'aqa-tests/TKG/lib'
+			timeout(time: 15, unit: 'MINUTES') {
+				copyArtifacts fingerprintArtifacts: true, projectName: "test.getDependency", filter: "json-simple.jar", selector: lastSuccessful(), target: 'aqa-tests/TKG/lib'
 			}
-		} catch (Exception e) {
-			echo 'Exception: ' + e.toString()
-			echo 'Cannot run copyArtifacts from test.getDependency. Skipping copyArtifacts...'
-		}
-
-		try {
-			if (env.BUILD_LIST.startsWith('system')) {
+			if (env.BUILD_LIST.contains('functional')) {
+				timeout(time: 60, unit: 'MINUTES') {
+					copyArtifacts fingerprintArtifacts: true, projectName: "test.getDependency", excludes: "json-simple.jar, jtreg*.*", selector: lastSuccessful(), target: 'aqa-tests/TKG/lib'
```

Comment:
I guess we can exclude jtreg from functional here, because its picked up in compile step, via the build.xml for the ssl-tests and CryptoTest.

Reply:
Yes, if the jtreg is not pre-downloaded, it will be downloaded via build.xml during the compilation stage. On second thought, since we are running functional test builds nightly, I think we should include jtreg (in dev and extended level only).

Reply:
PR is updated. [Grinder - extended.functional](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/36001/console)

<img width="703" alt="image" src="https://github.com/adoptium/aqa-tests/assets/19273206/b7761c81-3717-4261-9ca9-66d5a9859258">